internal/services/oracle/oracle_cloud_vm_cluster_data_source.go

```diff
@@ -211,6 +217,24 @@ func (d CloudVmClusterDataSource) Attributes() map[string]*pluginsdk.Schema {
 			Computed: true,
 		},
 
+		"file_system_configuration": {
+			Type:     pluginsdk.TypeList,
+			Computed: true,
+			Elem: &pluginsdk.Resource{
+				Schema: map[string]*pluginsdk.Schema{
+					"mount_point": {
+						Type:     pluginsdk.TypeString,
+						Computed: true,
+					},
+
+					"size_gb": {
```

Comment:
[ref](https://github.com/hashicorp/terraform-provider-azurerm/blob/main/contributing/topics/reference-naming.md?plain=1#L23)

```suggestion
					"size_in_gb": {
```